packages/react-server/core/renderMiddleware.js

```diff
@@ -43,18 +42,8 @@ var ELEMENT_ALREADY_WRITTEN = -2;
 /**
  * renderMiddleware entrypoint. Called by express for every request.
  */
-module.exports = function(server, routes) {
-
-	expressState.extend(server);
-
-	// parse cookies into req.cookies property
-	server.use(cookieParser());
-
-	// sets the namespace that data will be exposed into client-side
-	// TODO: express-state doesn't do much for us until we're using a templating library
-	server.set('state namespace', '__reactServerState');
-
-	server.use((req, res, next) => { RequestLocalStorage.startRequest(() => {
```

Comment:
Not a problem!

I added the `breaking change` label to this so that we remember to update our code internally when we merge it.